config/kubernetes/sig-docs/teams.yaml

```diff
@@ -342,6 +344,7 @@ teams:
     - SataQiu # L10n: Chinese
     - seokho-son # L10n: Korean
     - sftim # L10n: English
+    - tanjunchen # L10n: Chinese
```

Comment:
Please remove this line. Enough L10n team members have write access.

Reply:
@zacharysarah  done.